features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -33,28 +33,51 @@ float GetMipLevel(float2 coords, Texture2D<float4> tex)
 }
 
 #if defined(LANDSCAPE)
+#	define HEIGHT_POWER 4.0
+#	define INV_HEIGHT_POWER 0.25
 
-float GetTerrainHeight(PS_INPUT input, float2 coords, float mipLevels[6])
+float GetTerrainHeight(PS_INPUT input, float2 coords, float mipLevels[6], float blendFactor, out float pixelOffset[6])
 {
-	float height = 0.0;
 	if (input.LandBlendWeights1.x > 0.0)
-		height = TexColorSampler.SampleLevel(SampTerrainParallaxSampler, coords, mipLevels[0]).w * input.LandBlendWeights1.x;
+		pixelOffset[0] = pow(input.LandBlendWeights1.x, 1 + 1 * blendFactor) * (pow(TexColorSampler.SampleLevel(SampTerrainParallaxSampler, coords, mipLevels[0]).w, blendFactor * HEIGHT_POWER));
```

Comment:
Idk if it is faster but perhaps power the entire weights all at once rather than one by one? Will look cleaner too.